_posts/2020-10-13-community-call-dei.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Summary of October 2020 Community Call - US-RSE's Role in Promoting Diversity, Equity and Inclusion
+tags: [community call]
+posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill
```

Comment:
```suggestion
posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill, Vanessa Sochat
```

Reply:
oops I did the same thing, this one looks good!